opentraveldata/optd_airline_best_known_so_far.csv

```diff
@@ -558,6 +559,7 @@ air-jazz-aviation-v1^^^^JZA^QK^983^Jazz Aviation^Jazz Air^^^^^en|Jazz Aviation|=
 air-jeju-air-v1^^2005-01-01^^JJA^7C^806^Jeju Air^^^^^https://en.wikipedia.org/wiki/Jeju_Air^en|Jeju Air|^^air-jeju-air^1^
 air-jet2-v1^^2003-02-12^^EXS^LS^949^Jet2.com^^^^^https://en.wikipedia.org/wiki/Jet2.com^en|Jet2.com|^^air-jet2^1^
 air-jet-4-you-v1^^^^^8J^0^Línea Aerea Eco Jet^^^^^^en|Línea Aerea Eco Jet|^^air-jet-4-you^1^
+air-jetair-caribbean-v1^^2022-11-15^^4J^JRC^JetAir Caribbean^^^^^https://nl.wikipedia.org/wiki/Jetair_Caribbean^^^^1^
```

Comment:
There are 2 issues with that line:
- there is a missing caret right before the airline name
- the 2-letter and 3-letter codes are swapped